WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2054,3 +2054,19 @@ ChooserListViewCustomStyleContainer     = @PanelBorder@; @FixBoxModel@;
 MetaPropertyTable       =
 MetaPropertyTableCell   = padding:@SkinWrapperPadding@ @WidgetInternalPadding@; padding-top:0;
 
+#######################################################################
+#   Task List view
+#######################################################################
+
+TaskListViewAddNewTaskIcon                                  = display:inline-block; margin-right:12px;
+TaskListViewNewTaskBannerWrapper                            = @Text-hint@ border-bottom:1px solid @SeparatorColor@; @InactiveCursor@ @FontSize-big@ padding:16px 0 16px 6px;
+TaskListViewHeaderItemLabel                                 = @FontSize-slightly-big@ @Text-light@ padding: 10px 0px 10px 11px;
+TaskReadingPaneRight-ListViewItemCheckboxCell               = padding-left:10px;
+TaskReadingPaneRight-ListViewItem                           = height:80px;
+TaskReadingPaneRight-ListViewItemSubject                    = vertical-align:bottom;
+TaskReadingPaneRight-ListViewItemDate                       = align:right; padding-top:16px;
+TaskReadingPaneRight-ListViewItemTopRowContainer            = height:60%; margin:0;
+TaskReadingPaneRight-ListViewItemBottomRowContainer         = margin-top:-6px;
+TaskReadingPaneRight-ListViewItemBottomIconOuterWrapper     = align:right; padding-right:16px;
+TaskReadingPaneRight-ListViewItemBottomIconInnerWrapper     = margin-left:auto;
+TaskReadingPaneRight-ListViewItemCheckIcon                  = position:relative; top:38%;
```

Comment:
I tried using vertical-align, here.
But it was not working as expected.
May be because of nesting inside tr and td.
Using position and using top is also one way to align element in middle